plenty_api/api.py

```diff
@@ -626,6 +635,7 @@ def plenty_api_get_referrers(self,
 
     def plenty_api_get_items(self,
                              refine: dict = None,
+                             query: dict = {},
```

Comment:
What is the purpose of this query dictionary? You should adjust the documentation of a public method if you change its parameter set. The whole idea behind `refine` and `additional`, was to abstract the raw query into something more user-friendly. Is this some kind of alternative to that idea? 
Additionally, I find it strange that you introduce that new parameter for the `get` variations and items methods but not for manufacturers where you keep on working with the local query dictionary. This is again a reason why I think that commit descriptions are super useful as I am not sure if this solves a particular use case or if manufacturers should be adjusted as well?

Reply:
This shouldn't have been in the PR, I removed it.